### PR TITLE
CONFLUENT: Remove support-metrics-common

### PR DESCRIPTION
**Pull request by ijuma**
C3 no longer needs it (functionality removed):
https://github.com/confluentinc/blueway/pull/2015

ksqlDB imported the relevant code into their tree:
https://github.com/confluentinc/ksql/pull/4474

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by soondenana** [comment]
I see bunch of files gone. So I assume C3 now has its own version of it and no one needs this code anymore.

**Comment by ijuma** [comment]
@soondenana that's what the PR description says. :)

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -60,8 +60,6 @@ versions += [
   apacheda: "1.0.2",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  avro: "1.9.1",
```
Comment (soondenana) [inline gradle/dependencies.gradle]: Does this need to be taken out of `confluent-import-control.xml` too?

Comment (ijuma) [reply gradle/dependencies.gradle]: Good point, we can. remove that file. Done.

**Pull request merged**